kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/SectionSidePanel/index.vue

```diff
@@ -68,9 +86,17 @@
       });
 
       return {
+        route,
         canGoBack,
         showSidePanel,
         handleClosePanel,
+        editSectionLabel$,
+        sectionOrderLabel$,
+      };
+    },
+    data() {
+      return {
+        PageNames,
```

Comment:
We can return this `PageNames` from the setup method, so we dont have to add this data() method just for this.